pwintools.py

```diff
@@ -542,28 +539,33 @@ def __init__(self, cmdline, flags = 0, nostdhandles = False):
         if self._create_process() != 0:
             raise(ValueError("CreateProcess failed - Invalid arguments"))
         super(Process, self).__init__(pid=self.__pid, handle=self.__phandle)
-        if not (flags & CREATE_SUSPENDED):

```

Comment:
If it doesn't pose any compat issues, we should remove this check and wait_initialized function.

Reply:
IT used to raise an error. Might be a compat issue. I am investigating.

Reply:
```
[*] ReadProcessMemory: [WinError 299] Seule une partie d’une requête ReadProcessMemory ou WriteProcessMemory a été effectuée.
[*] Process <Process "hello" pid 78436 (DEAD) at 0x2d38ed4b640> not initialized ...
```

I think this is not only a compat issue but a system programming issue. In my humble opinion. This should be let like that.